modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/css/_general.scss

```diff
@@ -1,3 +1,11 @@
+.col-md-4 {
```

Comment:
Hi Aline! Just to confirm that the form widget in this case can't have a "lower" class like md-3 right? The site's page max division is three columns, so I think we can leave this way you did!

Reply:
Hi Rebeca! Yes, I considered the minimal size of site's page.

Reply:
That's great, thanks for answering!